modules/jira/widget.go

```diff
@@ -75,6 +75,9 @@ func (widget *Widget) Refresh() {
 		})
 	} else {
 		widget.result = searchResult
+		if len(widget.result.Issues) > 0 && widget.selected < 0 {
+			widget.selected = 0
```

Comment:
A few things, having recently dived into the selection code:
1. This means that every time the widget refreshes, it will try to highlight the first row (highlighting is broken right now, but should get fixed soonish)
1. Rather than starting with things not selected (default case right now, with selected defaulting to -1), we instead select the first row
In both the above cases, this is perhaps not a great user experience. By default, I think I prefer a stock list without highlighting (which can be cleared), vs having a row always highlighted
1. This also means that when things are refreshed, it will reset where the positioning was to the first item again. I can see this being frustrating if you are scrolling through and hit a refresh.

Reply:
@gnanderson Your thoughts on the comments above?